doc/source/development/contributing_code_style.rst

```diff
@@ -0,0 +1,199 @@
+.. _Not_sure_what_to_put_here:
+
+{{ header }}
+
+=======================
+pandas code style guide
+=======================
+
+.. contents:: Table of contents:
+   :local:
+
+Patterns
+========
+
+foo.__class__
+-------------
+
+*pandas* uses 'type(foo)' instead 'foo.__class__' as it is making the code more
+readable.
```

Comment:
Where are you getting this standards from? I don't see any of this in the wiki https://github.com/pandas-dev/pandas/wiki/Code-Style-and-Conventions, and I don't think this is an agreed practice, I usually use `foo.__class__.__name__`.

Reply:
https://github.com/pandas-dev/pandas/pull/29775#discussion_r349407678